src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -0,0 +1,249 @@
+import { FormikProps } from 'formik';
+import React, { useCallback, useState, useMemo } from 'react';
+import { defineMessages } from 'react-intl';
+import { ROOT_DOMAIN_ID, ColonyRole } from '@colony/colony-js';
+
+import { mergePayload, withKey, mapPayload, pipe } from '~utils/actions';
+import { ActionTypes } from '~redux/index';
+import { useTransformer } from '~utils/hooks';
+import { ItemDataType } from '~core/OmniPicker';
+import Heading from '~core/Heading';
+import Button from '~core/Button';
+import Dialog from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+import { SpinnerLoader } from '~core/Preloaders';
+import SingleUserPicker, { filterUserSelection } from '~core/SingleUserPicker';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import {
+  useColonyQuery,
+  useLoggedInUser,
+  useColonySubscribedUsersQuery,
+  useUser,
+  AnyUser,
+} from '~data/index';
+
+import {
+  getUserRolesForDomain,
+  getAllRootAccounts,
+  getAllUserRolesForDomain,
+} from '../../../transformers';
+import PermissionManagementForm from './PermissionManagementForm';
+import { availableRoles } from './constants';
```

Comment:
You don't really need this constants file anymore.

Just import the roles directly from `colonyJS`.

Eg:
```
import { ColonyRole } from '@colony/colony-js';
```

Reply:
Never mind about this, we discussed it privately and it can stay like it is